creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -0,0 +1,125 @@
+const moment = require('moment')
+
+const redisClient = require('../redis')
+const { logger: genericLogger } = require('../logging')
+
+const SYNC_STATES = Object.freeze({
+  success: 'success',
+  fail: 'fail'
+})
+
+// Make key expire in 7 days in seconds
```

Comment:
also this is meant to record daily sync volume right? seems to actually be recording weekly. or am i misreading

Reply:
this records the syncs as they appear daily. this is the ttl for each key, which is 1 week

Reply:
ahh i think i see - the redis keys are defined as:
```
// ex: aggregateSync:::05212021:::success
const prefix = `aggregateSync:::${new Date().toISOString().split('T')[0]}`
```
i presume from the example this means each day a new key is generated, so that makes sense

so what the TTL is doing here is actually ensuring that the data for the key is cleaned up after a week, meaning we only have access to a week's worth of data at any time right

Reply:
yes, this is correct. I did not think sync history over a week old was useful. however, can update this `EXPIRATION` key to a longer time frame. this week long duration was arbitrary

Reply:
yeah given how little data this is storing, i think its totally reasonable to set this to a larger value but its not really necssary

Reply:
discussed offline -- updating ttl to 90 days